.yamllint

```diff
@@ -17,3 +17,5 @@ ignore:
   - .tekton/
   - integration-tests/collectors/vault/managed-secrets.yaml
   - integration-tests/collectors/vault/tenant-secrets.yaml
+  - integration-tests/fbc-release/vault/managed-secrets.yaml
+  - integration-tests/fbc-release/vault/tenant-secrets.yaml
```

Comment:
Why this? I don't see `integration-tests/fbc-release` in this repo.

Reply:
yep. my bad. removed from this PR